tools/helper.psm1

```diff
@@ -0,0 +1,292 @@
+
+using namespace System.Runtime.InteropServices
+
+$IsWindowsEnv = [RuntimeInformation]::IsOSPlatform([OSPlatform]::Windows)
```

Comment:
This API might not be present on some systems, e.g. if someone is building on Win7.

FWIW, I planned on removing the dependency to this api completely and stop packaging `System.Runtime.InteropServices.RuntimeInformation.dll`. We only need this api for Windows vs. not and I feel confident there is a simple runtime test to know you're running on Windows, e.g. checking if $env:OS -eq 'Windows_NT' and maybe for extra robustness check $env:SystemRoot for a well known file.

Reply:
Yes, changed to `$IsWindowsEnv = [System.Environment]::OSVersion.Platform -eq "Win32NT"`